docs/rules/no-trailing-zeros.md

```diff
@@ -0,0 +1,48 @@
+---
+pageClass: "rule-details"
+sidebarDepth: 0
+title: "yml/no-trailing-zeros"
+description: "disallow trailing zeros for numbers"
+since: "v1.5.1"
+---
+
+# yml/no-trailing-zeros
+
+> disallow trailing zeros for numbers
+
+- :gear: This rule is included in `"plugin:yml/standard"`.
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule enforces the removal of unnecessary trailing zeros from numbers.
```

Comment:
It might be better to use the word `floats` rather than `numbers`.


https://yaml.org/spec/1.2.2/#:~:text=resolution%20of%20integers%2C-,floats,-%2C%20timestamps%20and%20similar